util/getBitness.ts

```diff
@@ -9,10 +9,8 @@ export const getBitness = async () => {
       'bitness',
     ]);
 
-    // Apparently in some cases this is not a Promise, then we should ignore
-    if (entropyValues instanceof Promise) {
-      return entropyValues.then(ua => ua.bitness);
-    }
+    // Apparently in some cases this is not a Promise, we can Promisify it.
+    return Promise.resolve(entropyValues).then(ua => ua.bitness);
```

Comment:
When the promise returns `undefined`, `ua.bitness` can throw an error

```suggestion
    return Promise.resolve(entropyValues)
      .then(({ bitness }) => bitness)
      .catch(() => undefined);
```

or we can wrap it with try catch

Reply:
> When the promise returns undefined, ua.bitness can throw an error

Right, that's what I was saying to @ljharb 👀